game/core/game_gui.py

```diff
@@ -0,0 +1,74 @@
+"""Game GUI."""
+
+import arcade
+from game.core.constants import SPRITE_SIZE
+
+
+class GameGUI():
+    """Model the Game's GUI."""
+    capacity = 10
+    vertical_hotbar_location = 40
+    hotbar_height = 80
+    sprite_height = 16
+    first_number_pad_sprite_index = 51
+    last_number_pad_sprite_index = 61
+    selected_item = None
+
+    def __init__(self, view):
+        self.window = view.window
+        self.player_sprite = view.player_sprite
+        self.selected_item = view.selected_item
```

Comment:
I think the `GameView.select_item` doesn't get updated and this isn't synced, but that's fine for now. An alternative might be to always pass selected_item from player_sprite in draw_inventory

Reply:
Fixed! It doesn't need to be passed around so I moved it to `game_gui`.